src/core/fuzz-unit-file.c

```diff
@@ -21,7 +21,11 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
         const char *name;
         long offset;
 
+        if (outside_size_range(size, 0, 65536))
```

Comment:
Looking at the parser it appears at list one corner case is no longer tested by the fuzzer because LONG_LINE_MAX is 1024 * 1024. I suspect most corner cases like that are no longer covered with those arbitrary limits in place which makes the fuzz targets less useful than they could probably be I think. I'm not sure those timeouts are worth it to be honest.

Reply:
I mean it should be possible to scale down those constants when the fuzz targets are built if the idea is to keep discovering corner cases without handling too much data but that's not what happened here and in PRs where the other arbitrary limits were introduced as far as I can see. I'm not sure how I missed them. Then again, those timeouts aren't exactly an issue that needs fixing I think